hazelcast/src/test/resources/test-hazelcast-real-jcache.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Seems yes - it uses only for this test as as I remember :) 

Do you mean remove all and keep only this part ? 

```
 <cache name="xmlCache*">
    </cache>
```